pkg/apis/pipeline/v1beta1/pipeline_validation.go

```diff
@@ -134,12 +134,22 @@ func (pt PipelineTask) Validate(ctx context.Context) (errs *apis.FieldError) {
 	if pt.Resources != nil {
 		errs = errs.Also(apis.ErrDisallowedFields("resources"))
 	}
-
+	// taskKinds contains the kinds when the apiVersion is not set, they are not custom tasks,
+	// if apiVersion is set they are custom tasks.
+	taskKinds := map[TaskKind]bool{
+		"":                 true,
+		NamespacedTaskKind: true,
+		"ClusterTask":      true,
```

Comment:
please use `ClusterTaskKind` constant 

https://github.com/tektoncd/pipeline/blob/79f6e4c48b1b815ec910da593a505db8f0a92b41/pkg/apis/pipeline/v1beta1/taskref_types.go#L54 

```suggestion
		ClusterTaskKind:      true,
```

Reply:
oh thanks!